string-manipulation-3/src/reverse-words.js

```diff
@@ -0,0 +1,13 @@
+/* exported reverseWords */
+/*
+purpose: write a function that reverses the words in a string while maintaining their
+order
+create a new function with parameter string
+split the string, reverse the values, join the string together
+then split the string at spaces, reverse the values, and join them together again
+
+*/
+
+function reverseWords(string) {
+  return string.split('').reverse().join('').split(' ').reverse().join(' ');
```

Comment:
This solution is pretty short, but it actually includes a total of **_6_** loops over the entire string. In terms of total operations, that is pretty intense. If you do this yourself by looping over the string, you can do this with one loop that includes one nested loop, and that's it 🤓